builder/googlecompute/step_create_image.go

```diff
@@ -58,7 +58,7 @@ func (s *StepCreateImage) Run(ctx context.Context, state multistep.StateBag) mul
 	}
 
 	if err != nil {
-		err := fmt.Errorf("Error waiting for image: %s", err)
+		err := fmt.Errorf("Error waiting for image: %s in", err)
```

Comment:
I presume you wanted to add the project ID in this log? If this is unaccessible I suggest we keep the previous message.

Reply:
Ah this changed when debugging intention was to remove this, have reverted it to the previous :)